src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,214 @@
+
+## Types
+
+The Stacks 2.0 supports a set of different transaction types:
+
+| **Type**          | **Description**                                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Coinbase          | The first transaction in a new block (an entity holding several transactions). Used to register for block rewards |
+| Token transfer    | Asset transfer from a sender to a recipient                                                                       |
+| Contract creation | Contract instantiation                                                                                            |
+| Contract call     | Contract call for a public, non read-only function                                                                |
+| Poison Microblock | Punish leaders who intentionally equivocate about the microblocks they package                                    |
+
+-> A sample of each transaction type can be found in the [Stacks Blockchain API response definition for transactions](https://blockstack.github.io/stacks-blockchain-api/#operation/get_transaction_by_id).
+
+## Post-conditions
+
+Transaction post-conditions are a feature meant to limit the damage smart contract bugs can do in terms of destroying a user's assets whenever a contract is instantiated or a public method of an existing contract is executed. Post-conditions are meant to be added by the user (or by the user's wallet software) at the moment they sign a transaction.
+
+Post-conditions are intended to be used to force a transaction to abort if the transaction would cause a principal to send an asset in a way that is not to the user's liking. For example, a user may append a post-condition saying that upon successful execution, their account's STX balance should have decreased by no more than 1 STX. If this is not the case, then the transaction would abort and the account would only pay the transaction fee of processing it.
+
+### Attributes
+
+Each transaction includes a field that describes zero or more post-conditions that must all be true when the transaction finishes running. A post-conditiona includes the following information:
+
+| **Attribute** | **Description**                                                                           |
+| ------------- | ----------------------------------------------------------------------------------------- |
+| Principal     | Sender of the transaction, can be a Stacks address or a contract                          |
+| Asset name    | Asset to apply conditions to (could be Stacks, fungible, or non-fungible tokens)          |
+| Comparator    | Compare operation to be applied (could define "how much" or "whether or not")             |
```

Comment:
Should we link to the spec or include some concrete examples of comparators or post-conditions?

Reply:
Good point! Let me add a column for an example and link to the principal doc